src/com/flower/kiev/Flower.java

```diff
@@ -0,0 +1,14 @@
+package com.flower.kiev;
+
+public abstract class Flower {
+
+    private final int FLOWER_PRICE;
```

Comment:
Why the name of the field in upper case? It is not right. It is not constant, so please use a camel style.
Modifier final just shows that this field would be initialized once. 
https://medium.com/@chuck.m.park/final-vs-static-final-in-java-cf73e6cc6848